phpBB/phpbb/db/migration/data/v33x/profilefield_youtube_update.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v33x;
+
+class profilefield_youtube_update extends \phpbb\db\migration\migration
+{
+	protected $youtube_url_matcher = 'https:\\/\\/(www\\.)?youtube\\.com\\/.+';
+
+	public function effectively_installed()
+	{
+		$profile_fields = $this->table_prefix . 'profile_fields';
+
+		$result = $this->db->sql_query(
+			"SELECT field_validation
+				FROM $profile_fields
+				WHERE field_name = 'phpbb_youtube'"
+		);
+
+		$row = $this->db->sql_fetchrow($result);
+		$this->db->sql_freeresult($result);
+
+		return $row['field_validation'] === $this->youtube_url_matcher;
+	}
+
+	static public function depends_on()
+	{
+		return [
+			'\phpbb\db\migration\data\v310\profilefield_youtube',
```

Comment:
I think this is not necessary.
`\phpbb\db\migration\data\v310\profilefield_youtube`

Reply:
Do you mean `v310\profilefield_youtube` isn't necessary because `v330\v330` is already included? If so, I'll remove it.

Reply:
Exactly, also `v330\v330` should be amended as `v330\v337`

Reply:
`return ['\phpbb\db\migration\data\v33x\v337'];`